code/game/machinery/bots/secbot.dm

```diff
@@ -384,6 +384,22 @@ Auto Patrol: []"},
 		target = user
 		mode = BOT_HUNT
 
+/obj/machinery/bot/secbot/Crossed(atom/movable/AM)
+	if(ismob(AM) && target)
+		var/mob/living/carbon/C = AM
+		if(!istype(C) || !C || in_range(src, target))

```

Comment:
If you want it to knock you down when beepsky hits you write a Cross() for beepsky.

Also Crossed() and Cross() are only called if you are on the target, you can't be next to something and call Crossed() or Cross() on it.